EZP-32198 - update Perso docs/add screenshots (#97)
ezsystems/user-documentation: docs/personalization/filters.md

|"Recommend items from the same main category and its subcategories" set to 2 category levels and below|Category "Desks/Tables" and below.|
|"Recommend items from the same main category and its subcategories" set to 3 category levels and below|Category "Tables" and below.|

You can provide multiple reference categories (both in the request and by defining 
a context item or items).
In such a case, the superset of the recommendations is returned, and the results 
are sorted based on a global weight of the recommendations. 
Depending on the popularity of the categories, the more popular categories push 
the less popular categories out of the results.

If the recommended item is located in more than one category, it is recommended 
that at least one category is requested in the recommendation call.

### Multiple category path dimensions for popularity models

The category path parameter is a powerful tool. 
A typical approach is to represent the default content in the navigation-based 
structure of a website. 
If you need to represent available items of a website in different dimensions (taxonomies), 

> you can do this out-of-the box, by enriching the `categorypath` information of an item.

Reviewer: ```suggestion
you can do this out of the box, by enriching the `categorypath` information of an item.
```

> For example, the shop mentioned above sells both furniture and plants, and products 

Reviewer: "shop mentioned above" - it either isn't mentioned, or too far away to refer to it

> For example, the shop mentioned above sells both furniture and plants, and products 

Author: ```suggestion
For example, in a shop that uses a category structure described in the preceding section to sell furniture
```

> are structured based on site navigation. 

Author: ```suggestion
and plants, products can be structured based on site navigation. 
```
